pkgcfg/package-configuration/postgresql.json

```diff
@@ -2,11 +2,17 @@
   "name": "postgresql",
   "version": "0.0.1",
   "match": "^postgresql(_[0-9]+)?$",
-  "readme": "To initialize the database run `initdb`.\nTo start the database run `pg_ctl -l .devbox/conf/postgresql/logfile start`.\nTo stop use `pg_ctl stop`.",
+  "readme": "To initialize the database run `initdb`.",
   "env": {
     "PGDATA": "{{ .DevboxRoot }}/conf/postgresql/data"
   },
   "create_files": {
     ".devbox/conf/postgresql/data": ""
+  },
+  "services": {
+    "postgresql": {
+      "start": "pg_ctl -l .devbox/conf/postgresql/logfile start",
```

Comment:
Should this be `conf/postgresql/logfile start`?

Reply:
The idea was to put non-shared files (logs, data, etc) in `.devbox` and put configs in `./config` but I do see how the naming is confusing. I'll rethink the naming.